docs/run_test/platform.rst

```diff
@@ -9,7 +9,69 @@ Run tests on different platforms
 Run on Azure
 ------------
 
-Follow the same procedure in :doc:`Getting started with Azure <quick_run>`.
+VM's can be deployed on Azure using images from vhd, shared image
+gallery or marketplace. If multiple types are specified, the first
+non-empty type is picked in the following order :
+vhd, shared image gallery and marketplace.
+
+Running using vhd
+^^^^^^^^^^^^^^^^^
+To run using vhd, add the following to runbook :
+.. code:: yaml
+   platform:
+   - type: azure
+      ...
+      requirement:
+         ...
+         azure:
+            ...
+            vhd: "<VHD URL>"
+
+The `<VHD URL>` can be a `SAS URL`.
+
+Running using marketplace
+^^^^^^^^^^^^^^^^^^^^^^^^^
+To run using marketplace image, add the following to runbook :
+.. code:: yaml
+   platform:
+   - type: azure
+      ...
+      requirement:
+         ...
+         azure:
+            ...
+            marketplace: "<Publisher> <Offer> <Sku> <Version>"
+
+Running using shared image gallery
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To run using shared image gallery, add the following
+to runbook if the shared image gallery is in the same
+subscription that is used to run LISA :
+.. code:: yaml
+   platform:
+   - type: azure
+      ...
+      requirement:
+         ...
+         azure:
+            ...
+            shared_gallery: "<image_gallery>/<image_definition>/<image_version>"
+
+If the shared image gallery is in a different subscription,
+`subscription_id` needs to be specified. Ensure that the
+authentication method has access to the shared image gallery.
+.. code:: yaml
```

Comment:
Not clear for 'authentication method' part

Reply:
The authentication method refers to managed identity/service principal/azure cli account having access to the shared image gallery. I didn't want to go into specifics of authentication methods used by LISA here as it is not relevant for this section. But please let me know if you have a better wording in mind to convey it clearly.